python/cog/json.py

```diff
@@ -0,0 +1,46 @@
+import json
```

Comment:
This needs tests. Maybe there are some we can pinch from keepsake?

We should also test different data types in end to end tests, because it touches Go and Python.